docs/source/getting-started/Setting-up-hypervisor-driver.adoc

```diff
@@ -152,5 +152,17 @@ $ sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve
 $ sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
 ----
 
-For more information, refer to the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
+For more information, see the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
 ====
+
+== Hyper-V
+To use Minishift with Hyper-V:
+
+. Install https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V].
+. Add a https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_virtual_switch[Virtual Switch].
++
+NOTE: Verify that you pair the virtual switch
+with a __network card (wired or wireless) that is connected to the network__.
```

Comment:
_connected to the network_ - I find this ambitious. To which _network_. My external LAN/WAN connection? Something which is connected to the internet?